client/app/components/Accordion/index.jsx

```diff
@@ -30,7 +30,10 @@ export const Accordion = ({
   const [open, setOpen] = useState(!!openProp);
 
   const displayContent = () => (
-    <div className={`${open ? 'accordionContent' : css.accordionClose}`}>
+    <div
+      className={`${open ? 'accordionContent' : css.accordionClose}`}
+      role="list"
```

Comment:
We should not be using the list role here. Let's use `role="region"` instead.

Reply:
1. Extending expect globally. After a quick research it turned out that `"@testing-library/jest-dom/extend-expect"` should be specified in package.json under `setupFilesAfterEnv` field. Then it worked as expected.

2. Sure, will do. Role adjusted to `region`